From 6a5eb1cff5a1e3c2499113ce4bc0a22cad14b4b8 Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Tue, 6 Sep 2011 23:37:29 +0900
Subject: [PATCH 1/2] Patch from Lars Wirzenius that

 * removes drivers from abstract (we're mentioned in acks, which is
   enough)
 * corrects a couple of missing spaces from markup
 * removes of useless appendix (this was in dep svn already, so better do
   it in the version in policy too)

http://bugs.debian.org/609160#109
---
 copyright-format/copyright-format.xml |   31 +++----------------------------
 1 files changed, 3 insertions(+), 28 deletions(-)

diff --git a/copyright-format/copyright-format.xml b/copyright-format/copyright-format.xml
index 8ae9023..d6790aa 100644
--- a/copyright-format/copyright-format.xml
+++ b/copyright-format/copyright-format.xml
@@ -30,9 +30,7 @@
         standard, machine-readable format for
         <filename>debian/copyright</filename> files within packages and
         facilitate automated checking and reporting of licenses for packages and
-        sets of packages. The DEP drivers were Steve Langasek
-        <email>vorlon@debian.org</email> and Lars Wirzenius
-        <email>liw@liw.fi</email>.
+        sets of packages.
       </para>
     </abstract>
   </articleinfo>
@@ -143,7 +141,7 @@
         A white space separated list means that the field value may be on one
         line or many, but values in the list are separated by one or more white
         space characters (including space, TAB, and newline).  For example, the
-        <varname>Files</varname>field has a list of filename patterns.
+        <varname>Files</varname> field has a list of filename patterns.
       </para>
     </section>
 
@@ -273,7 +271,7 @@
           package, for instance when a work combines a permissive and a copyleft
           license, or to document a <emphasis>compilation copyright</emphasis>
           and license.  It is possible to use only <varname>License</varname> in
-          the header paragraph, but <varname>Copyright</varname>alone makes no
+          the header paragraph, but <varname>Copyright</varname> alone makes no
           sense.
         </para>
       </section>
@@ -1171,27 +1169,4 @@ License: GPL-2+
     </example>
   </section>
 
-  <section id="appendix">
-    <title>
-      Appendix: Note about the use of this format in Debian
-    </title>
-    <para>
-      The Debian Policy (ยง<ulink
-      url="http://www.debian.org/doc/debian-policy/ch-docs#s-copyrightfile">12.5</ulink>)
-      demands that each package is accompanied by a file,
-      <filename>debian/copyright</filename> in source packages and
-      <filename>/usr/share/doc/package/copyright</filename> in binary packages,
-      that contains a verbatim copy of its copyright and distribution license.
-      In addition, it requires that copyrights must be extractable by mechanical
-      means.  This proposal for machine-readable copyright and license summary
-      files has been crafted for Debian's use, but it is our hope that other
-      software distributions, as well as upstream developers will adopt it, so
-      that review efforts can be easily reproduced and shared.
-    </para>
-    <para>
-      The copyright of the Debian packaging and the history of package
-      maintainers is simply indicated in a <literal>Files: debian/*</literal>
-      paragraph.
-    </para>
-  </section>
 </article>
-- 
1.7.5.4

